src/commands/secrets/upload.ts

```diff
@@ -38,13 +45,22 @@ export default class SecretsUpload extends BaseCommand {
 
   async run(): Promise<void> {
     const { flags, args } = await this.parse(SecretsUpload);
-    const account = await AccountUtils.getAccount(this.app, flags.account);
+    if (!flags.account) {
+      this.log(chalk.red('Account is not found. Please see examples below:'));
+      this.log(`  ${SecretsUpload.examples.join('\n  ')}`);
+      return;
+    }
 
+    const account = await AccountUtils.getAccount(this.app, flags.account);
     const is_admin = await UserUtils.isAdmin(this.app, account.id);
     if (!is_admin) {
       this.error('You do not have permission to upload secrets. Please contact your admin.');
     }
 
+    if (flags.platform && flags.environment) {
+      throw new Error('Please provide either the platform flag or the environment flag and not both.');
```

Comment:
This should be `ArchitectError` with `track` set to false.

Reply:
track if false by default now